src/themes/default/components/core/blocks/Header/CompareIcon.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <router-link :to="localizedRoute('/compare')" class="compare-icon no-underline inline-flex" v-if="isActive">
+  <router-link :to="localizedRoute('/compare')" class="compare-icon no-underline inline-flex" v-if="isActive" data-testid="comparelist-icon">
```

Comment:
please split comparelist to `data-testid="compare-list-icon"` or `data-testid="compare__list-icon"`. It's easier to read when there is separator